packages/xml-views-completion/test/providers/elementName/classes-spec.ts

```diff
@@ -360,6 +389,40 @@ describe("The ui5-language-assistant xml-views-completion", () => {
                   },
                 });
               });
+
+              it("will only suggest classes from the specified namespace and not its sub-namespaces", () => {
+                const xmlSnippet = `
+                  <mvc:View
+                    xmlns:mvc="sap.ui.core.mvc"
+                    xmlns:core="sap.ui.core"
+                    >
+                    <mvc:content>
+                      <core:⇶
+                    </mvc:content>
+                  </mvc:View>`;
+
+                testSuggestionsScenario({
+                  model: ui5Model,
+                  xmlText: xmlSnippet,
+                  providers: {
+                    elementName: [classesSuggestions],
+                  },
+                  assertion: (suggestions) => {
+                    assertSuggestionProperties(suggestions, "content");
+                    const suggestionNames = map(suggestions, (_) =>
+                      ui5NodeToFQN(_.ui5Node)
+                    );
+                    expect(suggestionNames).to.deep.equalInAnyOrder([
```

Comment:
I recommend also adding an **explicit** assertion on something that should not be present.
> expect(...).to.not.include.members(["subns-name1"]).

It does not add anything logically, but strengthens the "point" we are trying to prove n this test to a **human** reader.

Reply:
Done